daphne/src/vdaf/mod.rs

```diff
@@ -307,6 +319,18 @@ impl<'req> EarlyReportStateInitialized<'req> {
         };
         Ok(early_report_state_initialized)
     }
+
+    /// Turn this report into a rejected report using `failure` as the reason for it's rejection.
```

Comment:
Here again: Just want to double check that overwriting the failure reason in case it was already rejected is intended.